portfolio/src/main/webapp/style.css

```diff
@@ -59,8 +59,9 @@
 }
 
 .button_gray {background-color:darkgray;} /* Gray */
-.button_black {background-color: Black;} /* Gray */
+.button_black {background-color: Black;} /* Black */
```

Comment:
All of these comments aren't great style for exactly this reason. Favor descriptive identifiers (which you have done) to needless comments.

Comments should describe why you're doing something, not what you're doing.

Reply:
I would just kill all of them

Reply:
Ok, thanks.